scripts/call_monitor_submission_demo.py

```diff
@@ -0,0 +1,34 @@
+"""Creating and Running Monitor Submission Workflow."""
+from firecloud import api as fapi
+from firecloud import errors as ferrors
+
+# Adjustable Args*
+workspace_name = 'HCA_Optimus_Pipeline_MS'
+workspace_project = 'broad-firecloud-dsde'
+submission_id = '990e5930-a9b8-4f70-b220-11b7ffe702bb'
+workflow_repo = 'horsefish'
+workflow_name = 'monitor_submission'
+
+# Getting the json for the Monitor Submission Workflow
+workflow = fapi.get_workspace_config(workspace_project, workspace_name, workflow_repo, workflow_name)
+if workflow.status_code != 200:
+    print(workflow.content)
+    raise ferrors.FireCloudServerError(workflow.status_code, workflow.content)
+else:
```

Comment:
you don't need this else -- since you raise an error if it's not a success code, you can just continue on with code and you know it will only continue if you got a success code. (same thing for line 26)

Reply:
and line 32